io/plugins/eu.esdihumboldt.hale.io.jdbc/src/eu/esdihumboldt/hale/io/jdbc/constraints/DatabaseTable.java

```diff
@@ -36,14 +36,30 @@
 	// Database support.
 	private final boolean useQuote;
 
+	private final boolean isTable;
```

Comment:
The `DatabaseTableFactory`should be adapted to properly persist and restore the constraint information, with the new information added to the constraint class.